apps/openmw/mwmechanics/creaturestats.hpp

```diff
@@ -33,6 +33,23 @@ namespace MWMechanics
         MagicEffects mMagicEffects;
         Stat<int> mAiSettings[4];
         AiSequence mAiSequence;
+#ifdef ANDROID
```

Comment:
The order of fields is important? Why not simple reorganize them as you made for Android, for all platforms and delete `#ifdef`?